### ISSUE TITLE
Review from the 'Getting Started with JavaFX 11'

### ISSUE THREAD

**Issue by Naoghuman** [openjfx/openjfx-docs#36]
Hi there, 
first I want to ask you if I can contribute to your greate project?
I have read the tutorial and some points I notice some points aren't clear for me or where I thing why is that so?

So my suggestion is that I will add here for every point a commentary is that for your okay (or should I do a list from points which can then chopped?).

**Comment by Naoghuman** [comment]
Missing links in the first chapter in the pages:
* JavaFX and IntelliJ
* JavaFX and NetBeans
* JavaFX and Eclipse

Every chapter begins with:
* This section explains how to use Java 11 and JavaFX 11 from `XY`
where `XY` is the IDE name. Suggestion from me -> the name from the IDE should be link to the homepage or download page from the IDE:

In generell:
In the documentation from my open source projects (https://github.com/Naoghuman) I try to link keywords the first time they are written in a document chapter. For example keywords are: JavaFX, JDK, Maven, NetBeans, Gradle...

**Comment by jperedadnr** [comment]
Of course, you are more than welcome to contribute either by filing issues or raising PRs with your proposals. The latter is better :)

**Comment by Naoghuman** [comment]
Thx @jperedadnr for the welcome, 
its not better to fill first an issue , so we can discuss the topic and then I will do a PR?

Perhaps the topic dont need to fix or there are some suggestion about it which I can then integrate into the PR. Additional I can link the PR to the issue.

**Comment by jperedadnr** [comment]
Of course, each PR should be linked to an issue, that you can open first, so people now that it is already filed. 
And possibly it can be assigned to you, if you are going to work on it.
